app/src/main/res/values/strings.xml

```diff
@@ -66,6 +65,11 @@
     <string name="lbl_transaction_date">Transaction Date</string>
     <string name="lbl_added_to_wallet">Added to Wallet</string>
     <string name="lbl_transaction_image">Transaction Image</string>
+    <string name="add_transaction">Add Transaction</string>
+    <string name="lbl_which_wallet_do_you_want_to_add">Which wallet do you want to add?</string>
+    <string name="title_add_transaction">Add Transaction</string>
+    <string name="next">Next</string>
+    <string name="add">Add</string>
```

Comment:
Add prefix to this string names ex. btn_next or lbl_next depends on what it's use is

Reply:
Revised in 1d68f15a12935763089100ada272a41f8f3c8fe9